unlock-app/src/components/interface/buttons/lock/Verifiers.tsx

```diff
@@ -1,13 +1,10 @@
 import React from 'react'
-import { GrUserAdmin as UserAdmin } from 'react-icons/gr'
+import Svg from '../../svg'
 import Button from '../Button'
 
 const Verifiers = (props: any) => (
   <Button label="Verifiers" {...props}>
-    <UserAdmin
-      style={{ height: '15px', stroke: 'var(--gray)' }}
-      className="text-gray-400"
-    />
+    <Svg.Members name="Verifiers" />
   </Button>
 )
 
```

Comment:
Do we really need one more button? 

I would also use IconButton in the UI library for icon only buttons. It's not fully featured yet but we can modify it from a single place to include stuff like automatic tooltips.